briefcase/django.py

```diff
@@ -43,13 +43,7 @@ def install_support_package(self):
     def install_extras(self):
         # Install additional elements required for Django
         print(" * Installing extras...")
-        print(" * Installing NPM requirements...")
-        package_url = "https://raw.githubusercontent.com/pybee/toga/master/src/django/package.json"
```

Comment:
I wasn't sure if we should be using the toga-django install requirements, or a package.json like the source I was using from `togatest`. But it looks like you've added the package.json into the template, which works 👍